pkgs/tools/misc/abduco/default.nix

```diff
@@ -17,8 +23,34 @@ stdenv.mkDerivation rec {
   installFlags = [ "install-completion" ];
   CFLAGS = lib.optionalString stdenv.isDarwin "-D_DARWIN_C_SOURCE";
 
+  patches = [
+    # https://github.com/martanne/abduco/pull/22
+    (fetchpatch {
+      name = "use-XDG-directory-scheme-by-default";
+      url = "https://github.com/${owner}/${name}/commit/0e9a00312ac9777edcb169122144762e3611287b.patch";
```

Comment:
Please do not substitute the download URLs. That makes it harder to search for them and is unecessary